nb/BUILD.gn

```diff
@@ -1,82 +0,0 @@
-# Copyright 2021 The Cobalt Authors. All Rights Reserved.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-import("//starboard/build/platform_path.gni")
-
-static_library("nb") {
-  if (is_starboard) {
-    sources = [
-      "allocator.cc",
-      "allocator.h",
-      "first_fit_reuse_allocator.cc",
-      "first_fit_reuse_allocator.h",
-      "fixed_no_free_allocator.cc",
-      "fixed_no_free_allocator.h",
-      "pointer_arithmetic.h",
-      "reuse_allocator_base.cc",
-      "reuse_allocator_base.h",
-    ]
-
-    deps = [
-      "//starboard:starboard_headers_only",
-      "//starboard/common",
-    ]
-
-    # TODO: b/295615064 Remove nb platform specific dependency.
-    if (defined(has_nb_platform) && has_nb_platform &&
-        current_toolchain == starboard_toolchain) {
-      deps += [ "//$starboard_path/nb:nb_platform" ]
-    }
-  }
-}
-
-target(gtest_target_type, "nb_test") {
```

Comment:
Do all these get copied over ?


I see fixed_no_free_allocator_test in common/BUILD.gn

Reply:
Yes all got moved. first_fit_reuse is now internal and went in with the internal previous CL.